projects/koyo/index.js

```diff
@@ -80,6 +84,15 @@ const chainTreasury = (chain) => {
       transform
     );
 
+    await sumKoyoLPTokens(
```

Comment:
sorry, isnt this double counting? these assets are already counted as tvl

Reply:
I can see it on the our side but not really on any other protocols side. The way I saw it is the same way as other protocols holding Uniswap LP tokens.
Do you consider it double-counting on our (Koyo) side or it being included in other protocols?

Reply:
shibui treasury is fine, but not your own

Reply:
> shibui treasury is fine, but not your own

Fixed.